src/CMakeLists.txt

```diff
@@ -17,8 +18,12 @@ elseif(${SDFORMAT12_FOUND})
   set(SDFORMAT_PKGCONFIG_NAME "sdformat12")
   set(CMAKE_CXX_STANDARD 17)
   set(CMAKE_CXX_STANDARD_REQUIRED ON)
+elseif(${SDFORMAT15_FOUND})
+  set(SDFORMAT_PKGCONFIG_NAME "sdformat15")
+  set(CMAKE_CXX_STANDARD 17)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
 else()
-  message(SEND_ERROR "Could not find sdformat, sdformat9 or sdformat12 with pkg-config")
+  message(SEND_ERROR "Could not find sdformat, sdformat9, sdformat12 or sdformat15 with pkg-config")
 endif()
 
```

Comment:
All these pkg_cfg lines could be replaced by

```
pkg_search_module(SDFORMAT REQUIRED sdformat sdformat9 sdformat12 sdformat14 sdformat15)
```
Which (assuming CMake>=3.16) will set `${SDFORMAT_MODULE_NAME}` to the first found sdformat module.
The C++ standard still needs to be set, though.

Reply:
If not successful, the output would be:
```
    -- Checking for one of the modules 'sdformat;sdformat9;sdformat12;sdformat14;sdformat15'
    CMake Error at /usr/share/cmake-3.28/Modules/FindPkgConfig.cmake:906 (message):
      None of the required 'sdformat;sdformat9;sdformat12;sdformat14;sdformat15'
      found
```